src/DataHash.hs

```diff
@@ -0,0 +1,13 @@
+module DataHash (DataHash (..)) where
+
+import Data.Aeson (FromJSON, FromJSONKey, ToJSON (toJSON))
+import Data.Text qualified as Text
+import Servant.API.Generic (Generic)
+
+newtype DataHash = DataHash {dataHash :: Text.Text}
```

Comment:
Rename `dataHash` to `unDataHash` or `getDataHash`. `dataHash` sounds like constructor

Reply:
Changed to `unDataHash`